roles/utils_build_push_image/files/create-build-config.sh

```diff
@@ -0,0 +1,76 @@
+#! /bin/bash
```

Comment:
could you get rid of this file, and move everything in the ansible? templating will be more robust this way